plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py

```diff
@@ -325,8 +327,9 @@ def _get_task_instances_without_date(
 def _get_task_instances(
     since, dag_ids, quantity, dagbag, include_logs, include_xcom, session
 ):
+    query_fields = deepcopy(ETaskInstance.query_fields()).extend(EDagRun.query_fields())
```

Comment:
Replace the asterix ("*") with `deepcopy`? Can you elaborate? :)

Reply:
in py27 you cannot do this

```python
foo(*iterable, *iterable2)
```

therefore we need to construct one iterable with all query fields.

I use `deepcopy` for `ETaskInstance.quey_fields()` so I dont modify the original db model  instance with 

```python
ETaskInstnce.query_fields().extend(EDagRun.query_fields())
```

which could result in some strange and difficult to track bugs

Reply:
Resolved, great find!